Content.Shared/DeltaV/Salvage/Systems/MiningPointsSystem.cs

```diff
@@ -0,0 +1,121 @@
+using Content.Shared.Access.Systems;
+using Content.Shared.DeltaV.Salvage.Components;
+using Content.Shared.Lathe;
+using Robust.Shared.Audio.Systems;
+
+namespace Content.Shared.DeltaV.Salvage.Systems;
+
+public sealed class MiningPointsSystem : EntitySystem
+{
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly SharedIdCardSystem _idCard = default!;
+
+    private EntityQuery<MiningPointsComponent> _query;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        _query = GetEntityQuery<MiningPointsComponent>();
+
+        SubscribeLocalEvent<MiningPointsLatheComponent, LatheStartPrintingEvent>(OnStartPrinting);
+        Subs.BuiEvents<MiningPointsLatheComponent>(LatheUiKey.Key, subs =>
+        {
+            subs.Event<LatheClaimMiningPointsMessage>(OnClaimMiningPoints);
+        });
+    }
+
+    #region Event Handlers
+
+    private void OnStartPrinting(Entity<MiningPointsLatheComponent> ent, ref LatheStartPrintingEvent args)
+    {
+        var points = args.Recipe.MiningPoints;
+        if (points > 0)
+            AddPoints(ent.Owner, points);
+    }
+
+    private void OnClaimMiningPoints(Entity<MiningPointsLatheComponent> ent, ref LatheClaimMiningPointsMessage args)
+    {
+        var user = args.Actor;
+        if (TryFindIdCard(user) is {} dest)
+            TransferAll(ent.Owner, dest);
+    }
+
+    #endregion
+    #region Public API
+
+    /// <summary>
+    /// Tries to find the user's id card and gets its <see cref="MiningPointsComponent"/>.
+    /// </summary>
+    /// <remarks>
+    /// Component is nullable for easy usage with the API due to Entity<T> not being usable for Entity<T?> arguments.
```

Comment:
use `&lt;` for left angle brace and `&gt;` for right angle brace or else you get this peak documentation

![image](https://github.com/user-attachments/assets/4edc9b3e-8b37-4e64-a640-f8e38f4d3ccd)

Reply:
i love xml